requirements.txt

```diff
@@ -30,7 +30,7 @@ requests
 archspec
 
 # cryptography 3.4.0 no longer supports Python 2.7
-cryptography==3.3.2; python_version == '2.7'
+cryptography==41.0.3; python_version == '2.7'
```

Comment:
This will be a non-issue soon, since for EasyBuild 5.0 we don't need to list this indirect dependency anymore - see https://github.com/easybuilders/easybuild-framework/blob/5.0.x/requirements.txt